core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -672,11 +673,12 @@ private class MutationWriter {
     public MutationWriter(int numSendThreads) {
       serversMutations = new HashMap<>();
       queued = new HashSet<>();
-      sendThreadPool = context.threadPools().getPoolBuilder(this.getClass().getName())
+      sendThreadPool = context.threadPools().getPoolBuilder("tserver.batch.writer.send.pool")
```

Comment:
Is there a reason this pool name doesn't use `METRICS_POOL_PREFIX`?

Reply:
With 58cba6563c the prefix is added by the builder.